library/pipebio_client.py

```diff
@@ -15,30 +14,35 @@
 
 class PipebioClient:
     _session: Session
-    authentication: Authentication
     shareables: Shareables
     entities: Entities
     jobs: Jobs
     sequences: Sequences
+    user: Any
 
     def __init__(self):
         self._session = Session()
+        api_key = os.environ['PIPE_API_KEY'] if 'PIPE_API_KEY' in os.environ else None
+        if api_key is None:
+            print('PIPE_API_KEY required.')
+            print(f'PIPE_EMAIL={api_key}')
+            quit()
+
+        # Set Bearer token header with API KEY
+        self._session.headers.update({"Authorization": f"Bearer {api_key}"})
+
         base_url = 'https://app.pipebio.com'
-        self.authentication = Authentication(base_url)
         self.shareables = Shareables(base_url, self._session)
         self.entities = Entities(base_url, self._session)
         self.jobs = Jobs(base_url, self._session)
         self.sequences = Sequences(base_url, self._session)
+        self.user = self.get_user(base_url)
 
```

Comment:
Some of the tests needed the user to get at the org/owner id, so added a call to the /me endpoint to populate client.user